### DISCUSSION TITLE
Will being the last account to unfollow a project on LGTM.com stop that project from being indexed?

### DISCUSSION THREAD

**Discussion by JLLeitschuh** [github/securitylab#167] [General] [answered]
I currently have something like ~1.7k projects I'm following on LGTM.com because of some automated import scripts I wrote. I notice that LGTM.com doesn't index all projects on GH, just the ones that are followed. If I'm the last user to stop following a project on LGTM.com will that repository continue to be indexed or will the auto-index stop?

I'm asking because I'd like to unfollow many of the 1.7k projects I'm now following as the LGMT.com UI is currently pretty unusable/slow, but I want to be able to continue to query them for BB hunting by adding them under custom project lists.

TL;DR: If I'm the last to unfollow a GH project, will LGTM stop auto-indexing/scanning that project?

**Comment by jbj** [comment, marked as answer]
Short answer: no. We don't remove projects from LGTM when they lose followers, and the set of projects on LGTM is only vaguely related to the number of followers.
